src/msds510/test_util.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+from msds510 import util
+
+class UtilTest(unittest.TestCase):
+
+    def setUp(self):
+        self.dict_x = {'a': 1, 'b': 52, 'd': 6}
+        self.list_y = ['a', 'c', 'd']
+
+    def test_float_str(self):
+        self.assertEqual(util.to_int('4.4'), None)
+
+    def test_invalid_str(self):
+        self.assertEqual(util.to_int('invalid string'), None)
+
+    def test_int_input(self):
+        self.assertEqual(util.to_int('4'), 4)
+
+    def test_valid_dict_input(self):
```

Comment:
this works acceptably but can be a problem if there are many tests in the same method. Basically, if the results end up being similar for each test assertion, it can be difficult to determine in which assertion the test failed. However, people certainly do make tests like this, so it's not that big of a deal.